corehq/apps/app_manager/static/app_manager/js/forms/case_config_ui.js

```diff
@@ -1,4 +1,4 @@
-hqDefine('app_manager/js/forms/case_config_ui', function () {
+hqDefine('app_manager/js/forms/case_config_ui', ['hqwebapp/js/bootstrap3/main'], function (webappMain) {
```

Comment:
You don't need to do this, and I believe this change is going to trigger a test failure. App manager is the one significant part of HQ that still relies on script tags, not requirejs or webpack, so its js setup is unique (in a bad way).